From a4121f27d0b9a6f9f57ab93469eaee585e220e89 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 3 May 2016 14:06:18 -0500
Subject: [PATCH 1/3] use _DEFAULT_SOURCE instead of _BSD_SOURCE

_BSD_SOURCE and _SVID_SOURCE are deprecated aliases for
_DEFAULT_SOURCE.  If _DEFAULT_SOURCE is present we do not
issue a warning; the expectation is that the source is being
transitioned to use the new macro.
---
 mount.aufs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mount.aufs.c b/mount.aufs.c
index 09cf630..8196d2a 100644
--- a/mount.aufs.c
+++ b/mount.aufs.c
@@ -22,7 +22,7 @@
  */
 
 #define _XOPEN_SOURCE	500	/* getsubopt */
-#define _BSD_SOURCE		/* dirfd */
+#define _DEFAULT_SOURCE		/* dirfd */
 
 #include <sys/types.h>
 #include <dirent.h>
-- 
2.8.1

